### ISSUE TITLE
Scoreboard Direct URL Access

### ISSUE THREAD

**Issue by rootelement** [topcoder-platform/leaderboard-ui#72]
We need a way to be able to publicly distribute a scoreboard without allowing members to guess what other pages/scoreboards are in the app. The problem with how it is now is the user can back out to another page, or the index page, and see other scoreboards or winners podium pages. This was solved by adding a login, but then you can't share/socialize the scoreboard.

I don't know if it'd be:
`https://<baseurl>/<track>/<uuid>`
or
`https://<baseurl>/<uuid>`

Use case is, we should be able to:

- [X] Send Scoreboards to Screens in TCO
- [ ] Pull up Scoreboards unauthorized in Topcoder
- [ ] Share them on Social Media

**Comment by callmekatootie** [comment]
That would basically mean that the index pages should NOT be publicly accessible, but everything else should be... Correct?

**Comment by rootelement** [comment]
Yes, but don't start on it yet, we're just logging things that came out of TCO and will add more detail and prioritize them.
